srcpkgs/guile/template

```diff
@@ -1,20 +1,30 @@
 # Template file for 'guile'
 pkgname=guile
 version=2.2.7
-revision=1
+revision=2
 build_style=gnu-configure
-configure_args="--disable-static --disable-error-on-warning"
+configure_args="--disable-static --disable-error-on-warning --program-suffix=2"
 hostmakedepends="pkg-config texinfo"
 makedepends="gc-devel gmp-devel libatomic_ops-devel libffi-devel libltdl-devel
  libunistring-devel readline-devel"
-short_desc="A portable, embeddable Scheme implementation written in C"
+short_desc="Portable, embeddable Scheme implementation written in C"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-3.0-or-later, LGPL-3.0-or-later"
 homepage="http://www.gnu.org/software/guile"
 distfiles="${GNU_SITE}/${pkgname}/${pkgname}-${version}.tar.gz"
 checksum=44b4c5fbbe257ccdebea18420212c9b3e90c3c86a54920d8554039fc6769a007
 subpackages="libguile guile-devel"
 
+alternatives="
+ guile:guile:/usr/bin/guile2
+ guile:guild:/usr/bin/guild2
+ guile:guile-config:/usr/bin/guile-config2
+ guile:guile-snarf:/usr/bin/guile-snarf2
+ guile:guile-tools:/usr/bin/guile-tools2
+ guile:guile.1:/usr/share/man/man1/guile2.1
+ scheme:scheme:/usr/bin/guile2
+ scheme:scheme.1:/usr/share/man/man1/guile2.1"
```

Comment:
I know XBPS handles alternative -> actual file badly, I'd be worried about the opposite direction as well.

Reply:
Aren't most guile programs compatible with specific versions of guile?